vite.config.ts

```diff
@@ -24,7 +24,20 @@ const electronViteConfig: UserConfigExport = {
   }
 }
 
-export default defineConfig({
+// FIXME: Potentially publish this as a dedicated plugin, if other projects
+//        run into the same issue
```

Comment:
what does this mean? what is the issue this is fixing? would be nice to have that context so we know what to look for

Reply:
The issue this PR resolves is not Heroic-specific, it's something every Electron app will run into when trying to use the React dev tools. Because of that, every project will have to do something like this (adding a plugin to their bundler that adds the `<script>` tag). Thus, it might make sense to publish just this small Vite plugin separately (instead of defining it in this config file), so everyone else can just install it instead of essentially copy-pasting this code